COPY/etc/httpd/conf.d/manageiq-redirects-cockpit

```diff
@@ -0,0 +1 @@
+# Managed by cockpit-ws worker when enabled
```

Comment:
My suggestion is coming from consistency with the `-ui`, `-websocket`, and `-ws` redirects. Assuming you're using the same timing as those workers this really is not needed.